Rdmp.Core/Logging/Listeners/ToLoggingDatabaseDataLoadEventListener.cs

```diff
@@ -68,6 +68,7 @@ public virtual void OnNotify(object sender, NotifyEventArgs e)
                 case ProgressEventType.Debug:
                     break;
                 case ProgressEventType.Information:
+                case ProgressEventType.Success:
```

Comment:
I theres going to be a LOT of case statements like this you will need to update in RDMP to add a new value to this enum.  Also the order of the Enum is important there are code which says things like `if worst <= Warning` so creating a value of Success that is higher than Error as a raw value is very bad.  Adding the Success at the same level as Information will be a breaking change for plugins which will require full refresh so that they get the new enum value into the compiled dll (basically old dlls won't work with new RDMP after this change).